src/Faker/Provider/es_CO/Person.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Faker\Provider\es_CO;
+
+class Person extends \Faker\Provider\Person
+{
+    protected static $maleNameFormats = array(
+        '{{firstNameMale}} {{lastName}} {{lastName}}',
+        '{{firstNameMale}} {{firstNameMale}} {{lastName}} {{lastName}}',
+        '{{firstNameMale}} {{firstNameMale}} {{lastName}} {{lastName}}',
+        '{{titleMale}} {{firstNameMale}} {{lastName}}',
+        '{{titleMale}} {{firstNameMale}} {{lastName}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}} ',
+        '{{titleMale}} {{firstNameMale}} {{lastName}} {{lastName}} ',
+    );
+
+    protected static $femaleNameFormats = array(
+        '{{firstNameFemale}} {{lastName}} {{lastName}}',
+        '{{firstNameFemale}} {{firstNameFemale}} {{lastName}} {{lastName}}',
+        '{{firstNameFemale}} {{firstNameFemale}} {{lastName}} {{lastName}}',
+        '{{titleFemale}} {{firstNameFemale}} {{lastName}}',
+        '{{titleFemale}} {{firstNameFemale}} {{lastName}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}} ',
+        '{{firstNameFemale}} {{firstNameFemale}} {{lastName}} {{lastName}} ',
+    );
+
+    protected static $firstNameMale = array(
```

Comment:
Sorry, I don't know what you mean. May be is where are this formats come from? if yes, there are no official source for this, just is what it is.